scripts/helmer.sh

```diff
@@ -128,6 +148,8 @@ do_install() {
         helm upgrade -i --wait cert-manager jetstack/cert-manager -n cert-manager --create-namespace --set crds.enabled=true
     fi
 
+    kw_version ">=1.23" && create_kw_namespace_with_psa $NAMESPACE
```

Comment:
kw_version is shortcut which internally uses helm ls to determine current version.
This is not available before installing kubewarden.

```suggestion
    is_version ">=1.23" "${vMap[app]}" && create_kw_namespace_with_psa $NAMESPACE
```